ec2-spot-elastic-inference/ec2-spot-elastic-inference.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash -x
+
```

Comment:
Same over here , author and some documentation of what this file does will make supporting things easier in the future.